eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1917,15 +1918,22 @@ public function removeTranslation(ContentInfo $contentInfo, $languageCode)
             }
             // check if the specified translation is not the only one
             if (count($versionInfo->languageCodes) < 2) {
-                throw new BadStateException(
-                    '$languageCode',
-                    'Specified translation is the only one Content Object Version has'
-                );
+                $singleLangVersions[] = $versionInfo->versionNo;
+                continue;
```

Comment:
Personally, I think that this code will be more readable if we use `else` statement here instead of `continue`

```php
if (count($versionInfo->languageCodes) < 2) {
    $singleLangVersions[] = $versionInfo->versionNo;
}
else {
   $versions[] = $versionInfo->versionNo;
}
```

Reply:
We rather try to use return early approach trying to avoid too many nested blocks. I adapted this rule to `continue` here. 

To me it also separates validation from action on valid version, which is to add it to versions list. Note that above this there's also another similar block with `continue` which is part of validation, so changing the last one is IMHO inconsistent ([whole context](https://github.com/alongosz/ezpublish-kernel/blob/58ae7dc3f14a4226004b05fba2be61c309ff50ca/eZ/Publish/Core/Repository/ContentService.php#L1914-L1925)).

POV @andrerom? :)

Reply:
In this case it won't add a level and will make the code easier to read, so I agree with @adamwojs here